flybywiresim/aircraft
# feat(a380x/mfd): add ATCCOM MSG RECORD page

Fixes #[issue_no]

## Summary of Changes


Implement basic layout of MSG RECORD page to prep for ATSU integration.

## Screenshots (if necessary)



![image](https://github.com/user-attachments/assets/9ba67cd4-998a-49ff-bcc1-c6225f247522)
![image](https://github.com/user-attachments/assets/3ce2acfe-6900-4017-a5de-c370493dfd15)
![image](https://github.com/user-attachments/assets/efaccf4e-a497-4748-a994-56bad6589f31)


## References


![image](https://github.com/user-attachments/assets/af356a2b-5ab0-4b9c-b53d-aeb5cdf335b1)
![image](https://github.com/user-attachments/assets/568b6310-1ded-451b-98a3-913a5bb9a3db)
![image](https://github.com/user-attachments/assets/a9b212ef-b336-40c1-8b31-6898c9d59867)
![image](https://github.com/user-attachments/assets/d0cc813e-b06b-4f1c-8cea-1cb1f9010bca)




## Additional context



Discord username (if different from GitHub): Heclak

## Testing instructions

1. Load into A380 and power on the systems as per normal
2. Navigate to the MSG RECORD page under ATCCOM
3. Check that the message record page displays both buttons. Refer to screenshots for reference.
4. Check both ALL MSG and MONITORED MSGS buttons lead to the correct pages and have the "not yet implemented" overlay on them.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
